### ISSUE TITLE
CodeQL should scan C# (dotnet app)

### ISSUE THREAD

**Issue by ctlong** [cloudfoundry/cf-smoke-tests#81]
tldr; CodeQL scanning should include the C# code in `assets/dotnet_simple`.

When I [setup](https://github.com/cloudfoundry/cf-smoke-tests/commit/c93b732662a7a8f941407b8bda22fc3d73d5da77) the CodeQL scan for the first time it [failed](https://github.com/cloudfoundry/cf-smoke-tests/runs/7546274031?check_suite_focus=true#step:4:186) with the following error:
```
Error: Could not auto-detect a suitable build method
  Error: We were unable to automatically build your code. Please replace the call to the autobuild action with your custom build steps.  Failure invoking /opt/hostedtoolcache/CodeQL/0.0.0-20220623/x64/codeql/csharp/tools/autobuild.sh with arguments .
```

I'm not sure what the build steps are to get this scan working, so I decided to make an issue and come back to it later. In the meantime, I've [updated](https://github.com/cloudfoundry/cf-smoke-tests/commit/915af013de42140247272d7e9b8f2049af521f74) the scan to not include C#.

**Comment by oliver-heinrich** [comment]
Hi @ctlong, are you planning to work on this issue? If not should we close it?

**Comment by ctlong** [comment]
No, I'm okay to close it.

**Comment by oliver-heinrich** [comment]
Thanks for the info. Can you close the ticket. I cannot do it as I don't have approver permissions as of now.
